backend/src/server.js

```diff
@@ -20,28 +20,30 @@ const driver = getDriver()
 
 const createServer = options => {
   const defaults = {
-    context: async ({ request }) => {
-      const user = await decode(driver, request.headers.authorization)
+    context: async ({ req }) => {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-07-15T10:41:00Z" title="Monday, July 15th 2019, 12:41:00 pm +02:00">Jul 15, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -20,28 +20,30 @@ const driver = getDriver()
 
 const createServer = options => {
   const defaults = {
-    context: async ({ request }) => {
-      const user = await decode(driver, request.headers.authorization)
+    context: async ({ req }) => {
```


Can you explain this change?
I see cases where I would like to go through the code base and do the opposite where I spell it  out further, but I respect that others like to shorten variables, but if I lengthen and you shorten, we are kinda working against each other, or should we try to decide a standard and all stick by it?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-15T11:20:56Z" title="Monday, July 15th 2019, 1:20:56 pm +02:00">Jul 15, 2019</time>_

---


Same, same … 😂

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-15T15:45:37Z" title="Monday, July 15th 2019, 5:45:37 pm +02:00">Jul 15, 2019</time>_

---


Again, not my decision, but `apollo`'s decision. I would spell it `request` and so does `graphql-yoga`, but `apollo` uses `req`.

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-07-15T15:48:40Z" title="Monday, July 15th 2019, 5:48:40 pm +02:00">Jul 15, 2019</time>_

---


Read [this](https://github.com/prisma/graphql-yoga/issues/449#issuecomment-440595783) for the **why**.

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-16T07:06:50Z" title="Tuesday, July 16th 2019, 9:06:50 am +02:00">Jul 16, 2019</time>_

---


Thanks for letting us understand. 👍🏼